src/docs/product/releases/usage/sorting-filtering.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Sorting & Filtering
+sidebar_order: 20
+description: "Learn how to sort and filter releases."
+---
+
+<Note>
+
+This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+</Note>
+
+## Sorting Releases
+
+On the **Releases** page, you can use the "Sort By" dropdown to sort releases by:
+- Date Created
+- Total Sessions
+- [Active Sessions/Users](/product/releases/health/#active-sessionsusers)
+- [Crash Free Sessions/Users](/product/releases/health/#crash-free-sessionsusers)
+- Build Number
+- [Semantic Version](/platform-redirect/?next=/configuration/releases/%23bind-the-version)
```

Comment:
Unless we're expecting this to be absent, we can also use this form for the link:
< PlatformLink to="/configuration/releases/#bind-the-version">Semantic Versioning< /PlatformLink> 

I'm going to add a linking guide to our contributor guidelines as we have multiple methods to link, depending on the type of link we're intending to build.

Reply:
Note I've added spaces before the opening "<" in both PlatformLinks because otherwise GH doesn't display them.

Reply:
@matejminar - apologies, but my edit isn't correct here. I've been testing our multiple link types as I prepare the guidelines and the one you have created is the correct format for this use case.